src/dune_rules/buildable_rules.mli

```diff
@@ -31,6 +31,18 @@ type kind =
       ; empty_module_interface_if_absent : bool
       }
 
+(** temporary hacks until we remove preprocssing settings from the cctx *)
+val pp_spec :
```

Comment:
I don't understand why this function has to be exposed, I don't see it used outside `Buildable_rules`.

Reply:
Good catch, it must have been in some older iteration. I got rid of it.